groupnews-web/src/main/java/com/epam/internship/controller/UserController.java

```diff
@@ -30,4 +33,9 @@
 	public ResponseEntity<User> registerUser(@RequestParam String name, @RequestParam String email) {
 		return new ResponseEntity<User>(userService.createUser(name, email), HttpStatus.OK);
 	}
+
+	@GetMapping("users/{id}")
+	public ResponseEntity<User> getSingleUser(@PathVariable("id") Long id) throws NotFoundException {
+		return new ResponseEntity<User>(userService.getUserById(id), HttpStatus.OK);
```

Comment:
Why do you need ResponseEntity here? You can simple return the user without wrapping.